course-backend/course-content/apps/api/src/app/app.module.ts

```diff
@@ -3,6 +3,7 @@ import { QuestionManagementModule } from './question-management/question-managem
 import { TypeOrmModule } from '@nestjs/typeorm';
 import {ConfigModule, ConfigService} from "@nestjs/config";
 import {UsersModule} from "./users/users.module";
+import {AuthModule} from "./auth/auth.module";
```

Comment:
**issue:** please configure and use prettier to format your code. We can have a call if you want? If you want to try it yourself, in Webstorm File/Settingss=> search prettier => select the prettier from node_modules, select on Reformat and on Save => apply

Reply:
Format all your code under API afterwards